gitlab/utils.py

```diff
@@ -161,3 +161,28 @@ def warn(
         stacklevel=stacklevel,
         source=source,
     )
+
+
+def _validate_attrs(
+    data: Dict[str, Any],
+    attributes: types.RequiredOptional,
+    excludes: Optional[list] = None,
```

Comment:
Can the type-hint say what it is a list of?

Maybe it is a list of strings?
```
exclude: Optional[List[str]] = None,
```

Reply:
thank @JohnVillalovos  .. I was able to git pull from my repo and all is well. I don't know that we can type-hint that list. That's a question for @nejch .. He wrote that .. it could be strings .. I don't know if it could be anything else.

Reply:
yes, this would be a list of strings that are keys of attributes to exclude from building the list. :)

Reply:
done

Reply:
> done

I don't see it as done...

Reply:
forgot to push